.github/workflows/build.yml

```diff
@@ -21,11 +28,29 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-vagrant-
 
+    - name: Install libvirt
+      if: matrix.provider == 'libvirt'
+      run: |
+        brew install qemu libvirt
+        # disable unsupported security features
+        echo 'security_driver = "none"' >> /usr/local/etc/libvirt/qemu.conf
+        echo "dynamic_ownership = 0" >> /usr/local/etc/libvirt/qemu.conf
+        echo "remember_owner = 0" >> /usr/local/etc/libvirt/qemu.conf
```

Comment:
@fsimonis This needs some additional context in the comments

Reply:
This was macOS specific and is now gone